### DISCUSSION TITLE
Roles to be filled.

### DISCUSSION THREAD

**Discussion by ZhienWang** [HACC2021/MVP#25] [General]
Team Lead - JJ Foust
Tech Lead - Zhien, Tyler Chong,
DevOps - Zhien
Front-End- Declan, Adrian, Zhien
Back-End - Tyler, Zhien, {slot open},
Product Manager, Information & Intelligence Gathering, - Judy, Jose Carrillo
Presenter - Allan, Declan/JJ Foust
Score Keeper - Judy Nguyen, {slot open}
Videography - Vanessa Garcia, Adrian Castillo
Art/Icon/Design Assets - Vanessa Garcia, Adrian Castillo (edited)

**Comment by JJ-Foust** [comment]
I updated to match what is in slack. We still have 3 members that are not taking any roles and we still have some slots that are not filled. 

@judyn808 do you want to help me with score keeper and presentation? We can tag team this.

**Reply by judyn808** [reply]
I can help with score keeping (very subjective though yeah?) and building presentation, but I can't present. I'm traveling out of town on the 19th.

**Reply by JJ-Foust** [reply]
I can present. Just trying to give everyone that is not coding the ability to put their skills to use. 

Grading is subjective. I really just want to make sure we are hitting all categories.
